src/seedu/addressbook/commands/AddTagCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.Tagging;
+import seedu.addressbook.data.tag.UniqueTagList.DuplicateTagException;
+
+/** Adds a tag to a person in the AddressBook.
```

Comment:
Glad to see you implemented the hypothetical commands =).
Do remember to check the coding standards for javadoc, there shouldn't be text after `/**`.